src/pages/Admin/Sidebar/Sidebar.tsx

```diff
@@ -51,6 +51,6 @@ export default function Sidebar({
 }
 
 const linkClassName = createNavLinkStyler(
-  "flex items-center gap-2 py-3 px-5 font-bold text-sm hover:text-blue-d1 transition ease-in-out duration-300 aria-disabled:text-navy-l2 aria-disabled:pointer-events-none",
+  "grid grid-cols-subgrid col-span-2 items-center py-3 px-5 font-bold text-sm hover:text-blue-d1 transition ease-in-out duration-300 aria-disabled:text-navy-l2 aria-disabled:pointer-events-none",
```

Comment:
```suggestion
  "flex flex-col gap-2 items-center py-3 px-5 font-bold text-sm hover:text-blue-d1 transition ease-in-out duration-300 aria-disabled:text-navy-l2 aria-disabled:pointer-events-none",
```
<img width="274" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/b027fd51-f5a7-487d-924f-453d7126ceb3">

this is an overdue design detail (link text alignment - see admin designs) that wasn't implemented before because [subgrid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid) wasn't available back then